src/seq.rs

```diff
@@ -601,23 +654,41 @@ impl fmt::Debug for Event {
     }
 }
 
-/// Low level methods to set/get data on an Event. Don't use these directly, use generic methods on Event instead.
-pub trait EventData {
+/// Low level method to check if event type supports a given data type. Don't use directly, use generic methods on Event instead.
+pub trait HasEventData {
```

Comment:
I think I should also remove the implementation of `EventData` for `Vec<u8>`. The documentation should refer the user to `new_ext` and `get_ext` for event types that carry variable-length data. There shouldn't be two ways to do the same thing, especially when one is less efficient than the other.